client/package.json

```diff
@@ -15,7 +15,7 @@
         "moment": "^2.29.4",
         "moment-timezone": "^0.5.47",
         "msal": "^0.1.3",
-        "node-sass": "^8.0.0",
+        "node-sass": "^9.0.0",
```

Comment:
I assume you did but I didn't see it mentioned - does the site launch fine after a quick npm start? I recall node-sass being a particularly problematic library as far as versions go so just want to make sure it doesn't need any other config stuff to fix it.